tests/test_data/schwab_equity_award_v2.json

```diff
@@ -0,0 +1,144 @@
+{
+    "FromDate": "01/01/2020",
+    "ToDate": "12/23/2023",
+    "Transactions": [
+        {
+            "Date": "09/27/2023",
+            "Action": "Deposit",
+            "Symbol": "GOOG",
+            "Quantity": "13.6",
+            "Description": "RS",
+            "FeesAndCommissions": null,
+            "DisbursementElection": null,
+            "Amount": null,
+            "TransactionDetails": [
+                {
+                    "Details": {
+                        "AwardDate": "01/01/2019",
+                        "AwardId": "C987654",
+                        "VestDate": "09/25/2023",
```

Comment:
what date format on earth is this? ah yes, US...

Reply:
Yes ISO 8601 date formats would be clearer and safer but we'll have to work with what Schwab gives us!